pkgs/tools/networking/ofono/default.nix

```diff
@@ -46,6 +46,10 @@ stdenv.mkDerivation rec {
     "--enable-external-ell"
   ];
 
+  installFlags = [
+    "SYSCONFDIR=${placeholder "out"}/etc"
+  ];
+
   postInstall = ''
```

Comment:
We still need `configureFlags = [ "--sysconfdir=/etc" ]` instead of this. Otherwise, the `installFlags` are no-op.

Reply:
nix-build try to `mkdir -p /etc/ofono` and failed with the `configureFlags = [ "--sysconfdir=/etc" ]`.
<details>
<summary>output</summary>

```installing
install flags: SHELL=/nix/store/xvvgw9sb8wk6d2c0j3ybn7sll67s3s4z-bash-4.4-p23/bin/bash gsettingsschemadir=/nix/store/5lswng3mabl82ykhy83s3xsxcnclsi8k-ofono-1.32/share/gsettings-schemas/ofono-1.32/glib-2.0/schemas/ pkgconfigdir=/nix/store/k35cz4wimj7rjq2rj5rl05ynw8fcni92-ofono-1.32-dev/lib/pkgconfig m4datadir=/nix/store/k35cz4wimj7rjq2rj5rl05ynw8fcni92-ofono-1.32-dev/share/aclocal aclocaldir=/nix/store/k35cz4wimj7rjq2rj5rl05ynw8fcni92-ofono-1.32-dev/share/aclocal SYSCONFDIR=/nix/store/5lswng3mabl82ykhy83s3xsxcnclsi8k-ofono-1.32/etc install
make --no-print-directory install-am
/nix/store/xs38mci2dj4isxayifazwdma1dk9w9bl-coreutils-8.32/bin/mkdir -p ‘/nix/store/5lswng3mabl82ykhy83s3xsxcnclsi8k-ofono-1.32/sbin’
/nix/store/xvvgw9sb8wk6d2c0j3ybn7sll67s3s4z-bash-4.4-p23/bin/bash ./libtool   --mode=install /nix/store/xs38mci2dj4isxayifazwdma1dk9w9bl-coreutils-8.32/bin/install -c src/ofonod ‘/nix/store/5lswng3mabl82ykhy83s3xsxcnclsi8k-ofono-1.32/sbin’
libtool: install: /nix/store/xs38mci2dj4isxayifazwdma1dk9w9bl-coreutils-8.32/bin/install -c src/ofonod /nix/store/5lswng3mabl82ykhy83s3xsxcnclsi8k-ofono-1.32/sbin/ofonod
/nix/store/xs38mci2dj4isxayifazwdma1dk9w9bl-coreutils-8.32/bin/mkdir -p ‘/etc/ofono’
/nix/store/xs38mci2dj4isxayifazwdma1dk9w9bl-coreutils-8.32/bin/mkdir: cannot create directory ‘/etc/ofono’: Permission denied
make[2]: *** [Makefile:4175: install-dist_confDATA] Error 1
make[1]: *** [Makefile:4803: install-am] Error 2
make: *** [Makefile:4796: install] Error 2
builder for ‘/nix/store/y1w0h5gq0zhfj85d3pb4s903n1q8lync-ofono-1.32.drv’ failed with exit code 2
error: build of ‘/nix/store/y1w0h5gq0zhfj85d3pb4s903n1q8lync-ofono-1.32.drv’ failed
```
</details>

BTW, `strings result/bin/ofonod | grep /etc` still shows a hardcoded path to `/nix/store/.../etc/ofono` but it turned out to be a symlink to `/etc/ofono` and it works without this PR... 🤔 
... https://github.com/NixOS/nixpkgs/pull/109109 has fixed my problem.
Thanks anyway.

Reply:
Sorry, it should be lowercase `sysconfig` for Autotools, fixed in e69ba1da47292ed38ebf2183b9f1b1511f259d2d